Description: comment out tests that require test data
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/encoder_7.rs
+++ b/src/encoder_7.rs
@@ -65,7 +65,7 @@
     }
 }
 
-#[cfg(test)]
+/*#[cfg(test)]
 mod test {
     use std::io::{Read, Write};
 
@@ -113,4 +113,4 @@
         let decoded_data = String::from_utf8(decoded).unwrap();
         assert_eq!(decoded_data, test_data);
     }
-}
+} */
--- a/src/encoder_8.rs
+++ b/src/encoder_8.rs
@@ -71,7 +71,7 @@
     }
 }
 
-#[cfg(test)]
+/* #[cfg(test)]
 mod test {
     use std::io::{Read, Write};
 
@@ -124,4 +124,4 @@
         let decoded_data = String::from_utf8(decoded).unwrap();
         assert_eq!(decoded_data, test_data);
     }
-}
+} */
--- a/tests/reference_test.rs
+++ b/tests/reference_test.rs
@@ -1,4 +1,4 @@
-use std::{
+/* use std::{
     io::{Read, Write},
     path::PathBuf,
 };
@@ -398,4 +398,4 @@
     #[test] fn decompression_ppmd8_issue3_9() { test_decompression::<PPMd8>(ISSUE3_PATH, 9); }
     #[test] fn decompression_ppmd8_pg100_9() { test_decompression::<PPMd8>(PG100_PATH, 9); }
     #[test] fn decompression_ppmd8_pg6800_9() { test_decompression::<PPMd8>(PG6800_PATH, 9); }
-}
+} */
